chucks-pick3-web/assets/css/navbar.css

```diff
@@ -146,9 +150,194 @@
 .sub-nav-bar-container .sub-nav-bar-col-50.allow-hover:hover {
     background-color: green;
     cursor: pointer;
+    text-decoration: none;
 }
 
 .sub-nav-bar-container .sub-nav-bar-col-50.active {
     background-color: gray;
 }
 
+/* Responsive Set Up------- */
+
+
+
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+.mobile-nav-bar-container {
+    display: none;
+}
+
+
+
+@media only screen and (max-width: 760px) {
+
+.main-nav-bar-container{
+    display: none;
+}
+    .sub-nav-bar-container {
+        display: none;
+    }
+
+    .sub-nav-bar-title-container {
+        display: none;
+    }
+
+    .float-right {
+        float: right;
+    }
+
+.mobile-nav-bar-container{
+    display: block;
+    width: 100%;
+    height: 50px;
+/*expands to 100% width */
+}
+
+    .mobile-nav-bar-container .nav-bar-col-25.allow-hover:hover {
+        background-color: green;
+        cursor: pointer;
+    }
+    .mobile-nav-bar-container a:hover {
+        text-decoration: none;
+        color: black;
+    }
+
+
+.logo-container {
+    width: 20%;
+    height: 75px;
+
+/* stays at 20% width, takes on height of header-img */
+}
+
+.header-img-container {
+    width: 80%;
+    height: 75px;
+/* stays at 80% width */
+}
+
+
+
+[class*="nav-bar-col-25"] {
+width: 20%
+
+}
+}
+
+/*---Hamburger Menu----*/
+
+
+
+.side-nav-stick{
+    position: fixed;
+    top: 124px;
+}
+
+.sidenav {
+    display: inline-block;
+    height: 100%;
+    width: 0;
+    position: fixed;
+    z-index: 2;
+    right: 0;
+    background-color: rgba(0,0,0,0.85);
+    overflow-x: hidden;
+    transition: 0.5s;
+    padding-top: 60px;
```

Comment:
In general, a good rule of thumb not to do padding that is more than 25px.   If that is not enough padding, look for another solution with "CSS Rule attribute position" set to either relative or absolute and use "CSS Rule attribute top"

Reply:
Removed padding and completed alignment using "CSS Rule Attribute Top".